app/views/automated_tests/_test_runs.html.erb

```diff
@@ -1,4 +1,16 @@
+<% content_for :head do %>
+  <script>
+    document.addEventListener('DOMContentLoaded', () => {
+      makeTestScriptResultTable(document.getElementById('test_script_result_table'));
+    })
+  </script>
+<% end %>
+
+
 <h2><%= t('automated_tests.test_results') %></h2>
+
+
```

Comment:
See my above comment. In general, you should stop adding blank lines as a habit.